designguidelines.md

```diff
@@ -0,0 +1,15 @@
+These components are used as the building blocks for creating all types of forms. These forms should be able to be used on all devices ie desktop, mobile and tablet.
+
+## Form rules:
+
+*Labels* - Always label the form component ie a text input, select box to help the user understand it’s meaning.
```

Comment:
Is this meant to be a list of items? Github shows it as a paragraph of text. https://github.com/Financial-Times/o-forms/blob/7bf92adcb97af6f11c088b5cbadada3ab64337b7/designguidelines.md